Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -279,6 +351,271 @@ - (void)finalizeInit
     is3PIDBindingInProgress = NO;
 }
 
+- (void)updateSections
+{
+    NSMutableArray<Section*> *tmpSections = [NSMutableArray arrayWithCapacity:SECTION_TAG_DEACTIVATE_ACCOUNT + 1];
```

Comment:
We should avoid to declare a size. Specially if SECTION_TAG_DEACTIVATE_ACCOUNT moves.

Reply:
Actually it's just declaring the initial capacity go give NSMutableArray a hint of array size. We can still exceed it.